core/types.go

```diff
@@ -1,7 +1,45 @@
 package core
 
-// RelaySequences represents unrelayed packets on src and dst
-type RelaySequences struct {
-	Src []uint64 `json:"src"`
-	Dst []uint64 `json:"dst"`
+import (
+	clienttypes "github.com/cosmos/ibc-go/v7/modules/core/02-client/types"
+	chantypes "github.com/cosmos/ibc-go/v7/modules/core/04-channel/types"
+)
+
+// PacketInfo represents the packet information that is acquired from a SendPacket event or
+// a pair of RecvPacket/WriteAcknowledgement events. In the former case, the `Acknowledgement`
+// field becomes nil. In the latter case, `EventHeight` represents the smaller height between
+// the ones of the `RecvPacket` event and the `WriteAcknowledgement` event.
+type PacketInfo struct {
+	chantypes.Packet
+	Acknowledgement []byte             `json:"acknowledgement"`
+	EventHeight     clienttypes.Height `json:"event_height"`
+}
+
+type PacketInfoList []*PacketInfo
```

Comment:
If you expect the elements to be ordered in ascending order of packet sequence, could you write this as a comment?

Reply:
No, `PacketInfoList` is sorted in the order in which events occur.
I will comment this. Thank you.

Reply:
> No, PacketInfoList is sorted in the order in which events occur.
oh, sorry. You are right.

Reply:
Fixed: https://github.com/hyperledger-labs/yui-relayer/pull/90/commits/9980e1375724fad9397ced3dce0fc855fd894e54